### DISCUSSION TITLE
Frigate Proxy as Home Assistant Dashboard

### DISCUSSION THREAD

**Discussion by DrSpaldo** [blakeblackshear/frigate#3993] [Ask A Question]
Hi, I was wondering if anyone has a good solution for making the Frigate Proxy (on the left side bar) as a dashboard / tab up the top? (in Home Assistant)

**Comment by NickM-27** [comment]
Use an iframe pointing to it

**Reply by DrSpaldo** [reply]
> Use an iframe pointing to it

Thanks @NickM-27 , I've tried using the iframe - https://frigate.address/ccab4aaf_frigate-proxy/dashboard 

The problem is that it loads the full HA frame, including the side menu bar, so it is there twice :(
